openapi.yaml

```diff
@@ -960,3 +997,109 @@ components:
         approvingTeamDetails:
           type: string
           example: "DevRel"
+    SchemaRequest:
+      title: SchemaRequest
+      required:
+        - schemafull
+      type: object
+      properties:
+        req_no:
+          title: req_no
+          type: integer
+          format: int32
+        topicname:
+          title: Topic name
+          description: Kafka Topic name
+          type: string
+          example: "testtopic"
+        environment:
+          title: environment
+          description: Id of the environment
+          type: string
+          example: "3"
+        environmentName:
+          title: environmentName
+          description: Name of the environment
+          type: string
+          example: "DEV"
+        schemaversion:
+          title: schemaversion
+          description: SchemaRequest version
+          type: string
+          example: "1.0"
+        teamname:
+          title: Team name
+          description: Topic owner team name
+          type: string
+          example: "Infra"
+        teamId:
+          title: Team ID
+          description: Team identifier
+          type: integer
+          format: int32
+          example: 1010
+        appname:
+          title: App name
+          example: "App"
+          type: string
+        schemafull:
+          title: schemafull
+          maxLength: 2147483647
```

Comment:
Question: is this length a hard limitation of the HTTP protocol or something? Haha!

Reply:
It may be a max length based on the database or max size for schemas in general, that's what I guessed. But I'm not sure.